frontend/packages/dev-console/src/components/pipelines/pipeline-builder/PipelineBuilderForm.tsx

```diff
@@ -80,6 +79,13 @@ const PipelineBuilderForm: React.FC<PipelineBuilderFormProps> = (props) => {
     highlightedIds: selectedIds,
   };
 
+  const handleReset = React.useCallback(() => {
+    // Ensure that the sidebar was not rendered again when the user press 'Cancel'
+    setSelectedTask(null);
+    selectedTaskRef.current = null;
+    props.handleReset();
+  }, [props]);
```

Comment:
why not destructure `handleReset` as well? Now useCallback dependency is on all the props instead of just being on `handleReset`?
```suggestion
    setSelectedTask(null);
    selectedTaskRef.current = null;
    handleReset();
  }, [handleReset]);
```

Reply:
:+1: Done